<!--REQUEST_ID=d8799a08-d67b-4a81-bfa8-5361b5f39c45-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48301&attempt_id=4fdc714b-effe-414d-9397-b6a85099c78f)

Attempted: monero-gui

<details><summary>Partial log (click to expand)</summary><p>

```
                                               ^~~
src/libwalletqt/WalletManager.cpp:170:67: warning: unused parameter 'dst' [-Wunused-parameter]
 bool WalletManager::moveWallet(const QString &src, const QString &dst)
                                                                   ^~~
src/libwalletqt/WalletManager.cpp: In member function 'QString WalletManager::walletLanguage(const QString&)':
src/libwalletqt/WalletManager.cpp:176:54: warning: unused parameter 'locale' [-Wunused-parameter]
 QString WalletManager::walletLanguage(const QString &locale)
                                                      ^~~~~~
builder for '/nix/store/pqhaqma7fbhcyxbcn18kqkhvjqfflxil-monero-gui-0.12.3.0.drv' failed with exit code 2
error: build of '/nix/store/pqhaqma7fbhcyxbcn18kqkhvjqfflxil-monero-gui-0.12.3.0.drv' failed
```
</p></details>

--- a/pkgs/applications/altcoins/monero-gui/default.nix
+++ b/pkgs/applications/altcoins/monero-gui/default.nix
@@ -5,20 +5,20 @@
 , qtlocation, qtquickcontrols2, qtwebchannel
 , qtwebengine, qtx11extras, qtxmlpatterns
 , monero, unbound, readline, boost, libunwind
-, pcsclite, zeromq, cppzmq, pkgconfig
+, libsodium, pcsclite, zeromq, cppzmq, pkgconfig
 }:
 
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "monero-gui-${version}";
-  version = "0.12.3.0";
+  version = "0.13.0.2";
 
   src = fetchFromGitHub {
     owner  = "monero-project";
     repo   = "monero-gui";
     rev    = "v${version}";
-    sha256 = "1ry0455cgirkc6n46qnlv5p49axjllil78xmx6469nbp3a2r3z7i";
+    sha256 = "02419rqi3zfy6yjyw3b3gvkbxh2ypdfpijiiklc7lyblb9nnjr68";
   };
 
   nativeBuildInputs = [ qmake pkgconfig ];
@@ -28,8 +28,8 @@ stdenv.mkDerivation rec {
     qtdeclarative qtlocation qtquickcontrols2
     qtwebchannel qtwebengine qtx11extras
     qtxmlpatterns monero unbound readline
-    boost libunwind pcsclite zeromq cppzmq
-    makeWrapper
+    boost libunwind libsodium pcsclite zeromq
+    cppzmq makeWrapper
   ];
 
   patches = [

